### PR TITLE
feat: add synchronization for Kotlin/Groovy tabs

### PR DESCRIPTION
**Pull request by p7nov**

### CODE REVIEW

File: docs/topics/configure-build-for-eap.md [hunk 1]
```diff
@@ -74,20 +79,25 @@ Here is an example.
...
 ```
 
-```kotlin
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
 dependencies {
-    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesEapVersion%")
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:%coroutinesEapVersion%"
```
Comment (shadrina) [inline docs/topics/configure-build-for-eap.md:96]: Minor. Seems like we usually use single-quoted strings in Groovy samples. Here are no templates inside, so maybe it is worth changing.

File: docs/topics/gradle.md [hunk 2]
```diff
@@ -36,40 +41,50 @@ require the `kotlin-multiplatform` plugin. [Learn more about the plugin](mpp-dis
...
 ```
 
-```kotlin
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
 plugins {
-    kotlin("jvm") version "%kotlinVersion%"
+    id "org.jetbrains.kotlin.jvm" version "%kotlinVersion%"
```
Comment (shadrina) [inline docs/topics/gradle.md:83]: The same here

**Review by shadrina — APPROVED** [review]

File: docs/topics/configure-build-for-eap.md (hunk 1 above)
Comment (shadrina) [reply docs/topics/configure-build-for-eap.md:96]: I see double quotes were already there, so it is kind of out of this PR context :)

Comment (p7nov) [reply docs/topics/configure-build-for-eap.md:96]: Yes, diffs look weird here because I've swapped Groovy and Kotlin code.

**Pull request merged**